src/SDFileLogger.h

```diff
@@ -47,7 +47,7 @@ class SDFileLogger final : public LoggerBase
 		print("[%d ms] ", millis());
 	}
 
-	void begin(SdFs& sd_inst)
+	void begin(SdFs& sd_inst, const char filename_[13] = "log000.txt")
```

Comment:
A minor nit, but the filename parameter should not have the trailing underscore.  That was there before to denote that filename_ is a class member.  In this case, it is simply a parameter.